### ISSUE TITLE
Show old and new method?

### ISSUE THREAD

**Issue by janpio** [BuddyLReno/cordova-plugin-media-duration-demo#3]
Could you maybe show both the old and the new method to update the duration somehow? Can one play two audio files at the same time?

**Comment by BuddyLReno** [comment]
Yeah I can write something up. Can't play two different files but I can have both methods hooked up.  I have an idea how to display the differences.

**Comment by BuddyLReno** [comment]
@janpio hey! I updated the app to show both old and new methods.  I also explain in the readme why you would want to use the newer method as opposed to the old, with screenshots.

**Comment by janpio** [comment]
Nice!

Once the "timer method" initially even didn't show the actual duration - It only appeared after I pressed Pause and Play.

Interesting other bug: Hitting "Pause" while it is still loading will keep playing the file while "Play" is showing (doesn't matter for your demo app of course).
